src/main/java/tasktype/TaskList.java

```diff
@@ -0,0 +1,7 @@
+package tasktype;
+
+import java.util.ArrayList;
+
+public class TaskList extends ArrayList<Task> {
+    //no code needed
+}
```

Comment:
Try to avoid empty classes. Perhaps you can use methods related to TaskList in this class.